src/shared/Atlassian.Bitbucket/BitbucketHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Atlassian.Bitbucket.Cloud;
+using GitCredentialManager;
+
+namespace Atlassian.Bitbucket
+{
+    public static class BitbucketHelper
+    {
+        public static string GetBaseUri(ISettings settings)
```

Comment:
Would prefer this to take a `Uri` directly rather than rely on `ISettings.RemoteUri` to have been set.